docs/testing/releases/330.md

```diff
@@ -0,0 +1,51 @@
+## Testing notes and ZIP for release 3.3.0
+
+[![Create Todo list](https://raw.githubusercontent.com/senadir/todo-my-markdown/master/public/github-button.svg?sanitize=true)](https://git-todo.netlify.app/create)
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-gutenberg-products-block/files/5148289/woocommerce-gutenberg-products-block.zip)
+
+### Enhancements – Cart & Checkout   <!-- heading -->
+
+#### Express payment methods in Cart block (Apple Pay, Chrome Pay)  <!-- heading -->
+
+- [ ] This feature requires the following:
+  - [WooCommerce Stripe extension](https://woocommerce.com/products/stripe) installed and active.
+  - One or more express payment methods supported and enabled on the store and client system (OS, browser). Note that Chrome Pay and Apple Pay are not supported in all regions. Please refer to documentation for Apple Pay and Chrome Pay for more information.
+- [ ] Go to `Dashboard > WooCommerce > Settings > Payments` and activate `Stripe – Credit Card`.
+- [ ] Click `Manage` button for `Stripe – Credit Card`.
+- [ ] Ensure `Payment Request Buttons` checkbox is enabled, and click `Save changes`.
+- [ ] Add the Cart block to a page and publish the page (for example, the main store `cart` page).
+- [ ] In the editor, the Cart block should show a preview of express payment options (for example, Apple Pay button).
+- [ ] View store front end in a browser with an express payment method enabled and configured. For example Chrome or Safari.
+- [ ] Add some products to cart.
+- [ ] View cart. If express payment is supported and available, you should see relevant express payment buttons above the cart `Proceed to checkout` button.
+  - Note: if you don't see any express payment options, confirm that they are working in the Checkout block, product page, or shortcode checkout page. 
+  - The cart block should include the same express payment options as other store pages.
+
+### Bug Fixes – Cart & Checkout   <!-- heading -->
+
+#### Fix alignment of discounted prices in Cart block - sale prices right-aligned  <!-- heading -->
+
+- [ ] Add a product on sale to the Cart and visit the Cart block.
+- [ ] Verify regular and discounted prices are both aligned to the right (see screenshots above for comparison).
```

Comment:
> (see **screenshots** above for comparison)

We will need to add the screenshots here. :slightly_smiling_face:

Reply:
I've deleted the mention of screenshots instead 😄